src/BTCPayServer.Lightning.Common/ILightningClient.cs

```diff
@@ -8,6 +8,7 @@ namespace BTCPayServer.Lightning
     public interface ILightningClient
     {
         Task<LightningInvoice> GetInvoice(string invoiceId, CancellationToken cancellation = default);
+        Task<LightningInvoice> GetInvoice(uint256 paymentHash, CancellationToken cancellation = default);
```

Comment:
Added this to be able to test the invoice that was generated by a keysend payemnt.